lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -283,6 +283,18 @@
 		"ev_curtailment_frac": {
 			"description": "Fraction between 0 and 1 that denotes curtailment of EV charging load to better align EV charging with expected energy production from a solar PV system",
 			"type": "number"
+		},
+		"characterize_residential_buildings_from_buildstock_csv":{
+		"description" : "Enable UO and ResStock connection to characterize residential buildings by matching them (and their properties) with building from the BuildStock CSV.",
```

Comment:
For other residential -related field descriptions, we use "Optional for residential buildings" or "Required for residential buildings". For consistency, should we do something similar here?